annotate.el

```diff
@@ -115,6 +115,43 @@
   :type 'string
   :group 'annotate)
 
+(defcustom annotate-maximum-size-checksum 5000000
+  "Calculate checksum of  the current buffer only if  the size is
+less than this size (in characters)"
+  :type  'integer
+  :group 'annotate)
```

Comment:
Why did you add this constraint?

Reply:
Because, i think, if the file is very large the calculation could take a lot of time so maybe an user could just skips this step and save time even if they know that annotation could be "out of sync".

Reply:
I would suspect that md5 will never take remotely as long as Emacs will take to load the file in the first place. After all, md5 is routinely used to checksum gigabytes worth of data.

Did you have an actual example where big files were a problem? Otherwise, I'd say we go without the added complexity of this constraint (for now). We can still add them later if it turns out to be a problem for users.

If that's ok with you, of course.

Reply:
Hello!

i am fine with removing the constraint, i have made a (very naive) test and -for a 5 years old computer- a ~200Mb file takes about 0.4s to calculate the hash and a couple of second (very approximate estimation!) to open in emacs, i am OK to remove the size check as i think is unusual to have more than a couple of files so big and i bet modern system got better calculating hash.

Maybe we could inform (in the README, for example) the user that such calculation occurs because, for example, i use desktop mode that restores all the buffers opened in a previous emacs session and is not unlikely (at least for me :)) to get restored more than 20 buffer or so. Anyway i just pushed the fix with the check removed and we can save the README update for another PR, if needed.

Bye!
C.

Reply:
Alright!